src/ConfigParser.h

```diff
@@ -206,18 +194,22 @@ class ConfigParser
 
     static SBuf CurrentLocation();
 
-    /// \returns a non-empty token sequence scanned by strtokFile()
-    Configuration::Tokens ftokens(const char *description) { return Configuration::Tokens(strtokFile, description); }
-
-    /// \returns a non-empty token sequence scanned by RegexStrtokFile()
-    Configuration::Tokens fregexTokens(const char *description) { return Configuration::Tokens(RegexStrtokFile, description); }
-
-    /// \returns a non-nil token extracted by strtokFile() or throws
-    const char *ftoken(const char *description) { return *Configuration::Tokens(strtokFile, description).begin(); }
-
     /// \returns global LegacyParser
     static ConfigParser &Current();
 
+    // The methods below interpret the quoted tokens as filenames if the
+    // configuration_includes_quoted_values configuration parameter is set to 'off'.
```

Comment:
> The parameters("/path/filename") syntax does not work for ACLs.

This is true only if `configuration_includes_quoted_values` is off (default). 
So, for ACLs parsed with strtokFile(), we can get arguments from a file, specifying either "/path/filename" (if `configuration_includes_quoted_values` is off) and parameters("/path/filename") (if `configuration_includes_quoted_values` is on). There are few ACLs which have arguments parsed with NextToken(), for example `has` ACL. These support loading arguments from file only if `configuration_includes_quoted_values` is on.

This behavior becomes clear if we notice that strtokFile() passes control to NextToken() if RecognizeQuotedValues is true. On the other hand, "parameters" special parsing is activated in ConfigParser::TokenParse() only if RecognizeQuotedValues is also true.

Reply:
Your last comment appears to confirm the concern I documented in the change request -- this is not about a specific configuration_includes_quoted_values setting but about the overall ability to load parameters from external files (the setting just affects the syntax for specifying external parameter files).

> There are few ACLs which have arguments parsed with NextToken(), for example has ACL. These support loading arguments from file (via "parameters") only if configuration_includes_quoted_values is on.

Is there a fundamental reason why those few ACLs are special/different? Without researching this, I speculate that the developer used NextToken() instead of strtokFile() by accident or, at the very least, without fully understanding the difference between the two APIs. If there are no fundamental reasons to keep these ACLs special, we can adjust their parsing code to use the new APIs so that they become just like all the other ACLs. This does not have to happen in this PR -- these exceptional ACLs are not in the way of this PR changes, including this code documentation, right?

Reply:
Ok, I tried to adjust this comment at 6bd3618.

> Is there a fundamental reason why those few ACLs are special/different?

No, I don't see any reason.

> these exceptional ACLs are not in the way of this PR changes

Agreed, we can fix them afterwards.